nav2_behavior_tree/include/nav2_behavior_tree/bt_service_node.hpp

```diff
@@ -48,7 +48,7 @@ class BtServiceNode : public BT::ActionNodeBase
     const BT::NodeConfiguration & conf,
     const std::string & service_name = "")
   : BT::ActionNodeBase(service_node_name, conf), service_name_(service_name), service_node_name_(
-      service_node_name)
+      service_node_name), should_send_request_(true)
```

Comment:
```suggestion
      service_node_name)
```

We set it in the start of tick, this is not necessary